assignment-08.Rmd

```diff
@@ -1,80 +1,102 @@
-    geom_point() +
-    facet_grid( ~ Species)
-
-team_theme <- function() {
-    # Make all your edits here for the theme.
-    # The code below is an example, change them when creating your theme
-    theme(axis.line = element_line(color = "red"),
-          text = element_text(size = 8),
-          legend.text = element_text(size = 12, family = "serif")) # this is an example
-}
-p + team_theme()
-```
-
--   ggplot2 official documentation on all the possible theme items:
-    <https://ggplot2.tidyverse.org/reference/theme.html>
--   ggplot2 official documentation on modifying theme items:
-    <https://ggplot2.tidyverse.org/reference/theme_get.html>
--   Great online book with tutorial walking through how to create a new
-    theme:
-    <https://bookdown.org/rdpeng/RProgDA/building-a-new-theme.html#creating-a-new-theme>
--   ggthemes (package with even more pre-created themes) documentation
-    on what possible themes look like (can use this as inspiration, and
-    even copy if you want! But don't copy the themes entirely... you
-    want the theme to be your own):
-    <https://cran.r-project.org/web/packages/ggthemes/vignettes/ggthemes.html>
-
-3.  Once you've created the custom theme, decide on a team member that
-    will save this theme to their fork of your team's GitHub account.
-    Create a new file in the `R/` folder called `graph_theme.R`. Paste
-    the `team_theme` function code into this file. Commit this file and
-    submit a PR of this change to the main team repo. (0.5 marks)
-    
-    - URL of pull request:
+    -   Create the following headers (make sure to use Markdown headers
+        `#`): "Introduction to the project", "Description of the data",
+        "Team description", "Installation instructions" (*note*, this
+        section we will provide for you).
+
+    -   Fill out the "Introduction to the project" section, answering
+        these questions in a paragraph form (don't include these
+        questions in the section). What is your project about? What is
+        the goal? Why are you doing it?
+
+    -   Fill out the "Description of the data" section, and briefly
+        write down what the data is about, what are the variables you
+        think you'll use, how the data was collected, and how it will
+        answer your research questions. Include a reference of the
+        dataset if one is available, for instance:
+
+            Forstmann BU, et al. (2014) Data from: Multi-modal ultra-high resolution
+            structural 7-Tesla MRI data repository. Dryad Digital Repository.
+            (https://dx.doi.org/10.5061/dryad.fb41s)
+
+    -   Fill out the "Team description" section by writing down a brief
+        biography of each member (including what their skills are and
+        what their approximate responsibilities (which can change later)
+        are for the project) and how team roles will be rotated.
+
+    -   URL of team member's pull request for `README.md`:
```

Comment:
Should we make each of them do an individual pull request for their bio section? Or not, in case it generates conflicts?